apps/cwl_to_slack/Makefile

```diff
@@ -1,24 +1,35 @@
-TARGET := cloudwatch-slack-notifications_App.zip
+ZIP_FILE := cloudwatch-slack-notifications_App.zip
+
+.PHONY: install
+install:
+	virtualenv -p python3 venv
+	. venv/bin/activate && pip install -r requirements.txt
 
-.PHONY: clean
-clean:
-	rm -rf $(TARGET)
 
 .PHONY: build
 build:
 	rm -rf target
 	mkdir target
-	npm install
-	cp *.js target/
-	cp package-lock.json target/
+	cp *.py target/
+	. venv/bin/activate && pip install -r requirements.txt -t target/ --upgrade
```

Comment:
if you run `venv/bin/pip`, you can skip the activate boilerplate,.

Reply:
thanks!

Reply:
👍